### PR TITLE
Base (finished)

### PR DESCRIPTION
**Pull request by legalsylvain**

### CODE REVIEW

**Comment by StefanRijnhart** [comment]
One pep8 item left: openerp/addons/base/migrations/8.0.1.3/pre-migration.py:27:1: E303 too many blank lines (3)

**Comment by bwrsandman** [comment]
:+1:

File: openerp/addons/base/migrations/8.0.1.3/pre-migration.py [hunk 1]
```diff
@@ -22,15 +22,13 @@
 
 from openerp.openupgrade import openupgrade
 
-column_renames = {
-    'res_company': [
-        ('paper_format', 'rml_paper_format'),
-    ]
-}
```
Comment (bwrsandman) [inline openerp/addons/base/migrations/8.0.1.3/pre-migration.py]: Just remove a line here.
The merge process seems to be adding a blank line, resulting in 3

**Comment by legalsylvain** [comment]
Close due to #60 Pull Request.

**Pull request closed without merging**